src/charactersheet/utilities/notifications.js

```diff
@@ -99,18 +99,14 @@ export var Notifications = {
         toggle: new Signal()
     },
 
-    abilityScores: {
-        changed: new Signal(),
-        intelligence: {
-            changed: new Signal()
-        },
-        dexterity: {
-            changed: new Signal()
-        }
+    abilityscore: {
```

Comment:
Though I agree that camelCase is more readable, I did the all lower because it allowed me to set up automatic subscriptions in the abstract classes. See ~line 140 in viewModels/abstract/tabular-view-models.js for an example.